### ISSUE TITLE
Improve the behavior of the mesage Optimize for screen reader on the home screen

### ISSUE THREAD

**Issue by llemeurfr** [edrlab/thorium-reader#3772]
On my device, a helper tool is using the same Web API as accessibility tools; therefore, Thorium incorrectly detects a screen reader. I must not be the only person with this kind of helper tool. 

As a consequence, the message "Optimize for screen reader (No) [SHIFT CTRL F12]" appears at the bottom of the home screen, as a link. If I click the link, I see a toast message: "Optimize for screen reader (Yes) [SHIFT CTRL F12]". If I go to the settings, I see that "Enhance screen reader experience" is now checked. Since I don't use a screen reader, the toast messages that appear while I'm reading are annoying.
If I go to the settings and deactivate the option, the message reappears at the bottom of the home screen, which is also annoying. 

We cannot make Thorium detect that not all helper tools are screen readers. 

Proposed solution:
- Keep the "Optimize for screen reader" message, because it attracts the attention of users who need it
- Remove the "(No) [SHIFT CTRL F12]" part, because this message never appears with a yes at this place, and the syntax is unclear; because users who need control keys get a complete list elsewhere; and because  [SHIFT CTRL F12] does not work on a Mac and it is complex to adapt the message to the OS. 
- Change the toast message for "Optimized for screen reader".
- When the user goes to the settings and deactivates the option, make sure via a persistent property that the message does not reappear at the bottom of the home screen.

**Comment by danielweck** [comment]
Related: https://github.com/edrlab/thorium-reader/issues/3516

Related: https://github.com/edrlab/thorium-reader/issues/3349


<img width="471" height="212" alt="Image" src="https://github.com/user-attachments/assets/46c7cd1d-434d-4518-b8a4-b7a4af13fd6f" />

**Comment by danielweck** [comment]
Technical hooks:

`app.isAccessibilitySupportEnabled()` - `app.setAccessibilitySupportEnabled(enabled)` ... and more interestingly `app.getAccessibilitySupportFeatures()`:

https://www.electronjs.org/docs/latest/api/app#appisaccessibilitysupportenabled-macos-windows

```js
  if (app.getAccessibilitySupportFeatures().includes('screenReader')) {
    // do stuff
  }
```

* `nativeAPIs` - Native OS accessibility APIs integration enabled.
* `webContents` - Web contents accessibility tree exposure enabled.
* `inlineTextBoxes` - Inline text boxes (character bounding boxes) enabled.
* `extendedProperties` - Extended accessibility properties enabled.
* `screenReader` - Screen reader specific mode enabled.
* `html` - HTML accessibility tree construction enabled.
* `labelImages` - Accessibility support for automatic image annotations.
* `pdfPrinting` - Accessibility support for PDF printing enabled.

Also see:

https://www.electronjs.org/docs/latest/api/app#appaccessibilitysupportenabled-macos-windows

https://www.electronjs.org/docs/latest/tutorial/accessibility

Historical issue: https://github.com/edrlab/thorium-reader/issues/3106

**Comment by danielweck** [comment]
Screen reader detection fixed via https://github.com/edrlab/thorium-reader/commit/c8839f213114ec9537225d7cb4c6866689c34629

Please test the MacOS ARM64 AppleSilicon build once it is ready :)

https://github.com/edrlab/thorium-reader/actions/runs/31727142851

https://github.com/edrlab/thorium-reader/releases/tag/latest-macos-arm
